src/webview/js/js.js

```diff
@@ -623,7 +614,9 @@ documentBody.addEventListener('click', (e: MouseEvent) => {
   }
 });
 
-const handleLongPress = (target: Element) => {
+const handleClicksOnMessage = (target: Element) => {
```

Comment:
This name is inaccurate:
* This isn't specific to messages -- notice the `target: 'header'` case.
* The `click` handler, above, is mostly about messages, so "on message" doesn't describe the distinction between them at all.

Reply:
I agree. At this point it seems better just to let it be `handleLongPress`.